includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -1920,12 +1976,21 @@ private function check_attr_spec_rule_value_properties( DOMElement $node, $attr_
 					$prop_value     = (float) $prop_value;
 				}
 				if ( isset( $required_value ) && $prop_value !== $required_value ) {
-					return AMP_Rule_Spec::FAIL;
+					return [
+						AMP_Rule_Spec::FAIL,
+						[
+							self::MISSING_REQUIRED_PROPERTY_VALUE,
+							[
+								'property' => $prop_name,
+								'value'    => $required_value,
```

Comment:
What about `property_name` and `property_value`? Or just `name` and `value`?

Reply:
`name` and `value` it is!